bindings/protocols/bacnet/index.html

```diff
@@ -998,41 +1006,16 @@ <h2>Examples</h2>
             "type": "number",
             "readOnly": true,
             "uriVariables": {
-                "covIncrement": {
+                "observeIncrement": {
                     "@type": "bacv:covIncrement",
                     "type": "number",
                     "minimum": 0
                 }
             },
             "forms": [{
                 "op": [ "observeproperty" ],
-                "href": "bacnet://5/0,1/85",
-                "bacv:usesService": "subscribeCOVproperty",
-                "bacv:hasDataType": {
-                    "@type": "bacv:Real"
-                }
-            }]
-        }
-    }
-}
-</pre>
-
-<pre id="example-form-variables" class="example" title="uriVariables in the forms element">
-{
-    "@context": ["https://www.w3.org/2022/wot/td/v1.1",
-    {
-        "bacv": "https://example.org/bacnet"
-    }],
-    ...
-    "properties": {
-        "analog1": {
-            "type": "number",
-            "readOnly": true,
-            "forms": [{
-                "op": [ "observeproperty" ],
-                "href": "bacnet://5/0,1/85",
-                "bacv:usesService": "subscribeCOVproperty",
-                "bacv:covIncrement": 0.2,
+                "href": "bacnet://5/0,1/85?covIncrement={observeIncrement}",
```

Comment:
It should actually look a bit different I am not wrong. Following https://www.w3.org/TR/wot-thing-description11/#example-32 , this kind of URL should look like `"bacnet://5/0,1/85/{?observeIncrement}"` . If you want to make sure we see `covIncrement` in the URI, we may need another approach.

Reply:
Example 32 in the TD spec isn't definitive in terms of the format of the URI variable query string format. RFC 3986 should be the definitive reference. https://datatracker.ietf.org/doc/html/rfc3986#section-3.4 It only says a query string can be a sequence of pchar (character set) and some additional characters you can use as delimiters, etc.

It is typical to format query string components with <identifier>=<value>, to allow for multiple parameters to be set explicitly. In the absence of a format mechanism, <identifier>=<value> is probably a good default mechanism. The URI Template should provide a way to describe the expected format of the query string, but URI templates are not well understood.

The uriVariable section is one I think needs a lot of work in TD.next, in conjunction with conceptualizing the parameterization of Property affordances, in general.

Reply:
TD Call of 22.11: My comment is addressed since I was referring to compact form without realizing. Proposal is the long syntax, which is also fine.

Reply:
I updated Chapter 4 now.